source/extensions/transport_sockets/proxy_protocol/proxy_protocol.cc

```diff
@@ -36,6 +40,11 @@ UpstreamProxyProtocolSocket::UpstreamProxyProtocolSocket(
       pass_through_tlvs_.insert(0xFF & tlv_type);
     }
   }
+  for (const auto& entry : config.added_tlvs()) {
+    added_tlvs_.emplace_back(Network::ProxyProtocolTLV{
+        static_cast<uint8_t>(entry.type()),
+        std::vector<unsigned char>(entry.value().begin(), entry.value().end())});
```

Comment:
Why are you constructing another copy of the vector here? Can't you just pass `envoy.value()`? Maybe you need `push_back` instead of `emplace_back`?

Reply:
Addressed in https://github.com/envoyproxy/envoy/pull/37591/commits/ad46d237feaee356618a904f327601d8f5efac1d.